NEWS.md

```diff
@@ -1,4 +1,17 @@
-## 4.6.3 2024-03-04
+
+## 4.6.1 2024-05-23
+[Full Changelog](https://github.com/folio-org/edge-common/compare/v4.6.0...v4.6.1)
+
+## Stories
+* [EDGCOMMON-79](https://folio-org.atlassian.net/browse/EDGCOMMON-79) - Add ssl support
```

Comment:
here we need to specify https://folio-org.atlassian.net/browse/EDGCOMMON-77 
79 is duplicate

Reply:
Fixed